File: Example/ExampleObjectiveC/EventViaNotification+CoreDataProperties.m
//
//  EventViaNotification+CoreDataProperties.m
//  ExampleObjectiveC
//
//  Created by zee-pk on 17/11/2016.
//  Copyright (c) 2016 predict.io by ParkTAG GmbH. All rights reserved.
//

#import "EventViaNotification+CoreDataProperties.h"

@implementation EventViaNotification (CoreDataProperties)

+ (NSFetchRequest<EventViaNotification *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"EventViaNotification"];
}

@dynamic accuracy;
@dynamic latitude;
@dynamic longitude;
@dynamic mode;
@dynamic timeStamp;
@dynamic type;
@dynamic zoneType;
@dynamic stationary;

@end

File: ExampleSwift/ExampleSwift/EventViaNotification+CoreDataProperties.swift
//
//  EventViaNotification+CoreDataProperties.swift
//  ExampleSwift
//
//  Created by zee-pk on 16/11/2016.
//  Copyright (c) 2016 predict.io by ParkTAG GmbH. All rights reserved.
//

import Foundation
import CoreData


extension EventViaNotification {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<EventViaNotification> {
        return NSFetchRequest<EventViaNotification>(entityName: "EventViaNotification");
    }

    @NSManaged public var accuracy: NSNumber?
    @NSManaged public var latitude: NSNumber?
    @NSManaged public var longitude: NSNumber?
    @NSManaged public var mode: NSNumber?
    @NSManaged public var timeStamp: NSDate?
    @NSManaged public var type: NSNumber?
    @NSManaged public var stationary: NSNumber?
    @NSManaged public var zoneType: NSNumber?

}
